### PR TITLE
Exclude empty targets, also fix ci-test-infra-bazel job

### PR DESCRIPTION
**Pull request by krzyzacy**
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/50333/pull-kubernetes-bazel/34672/

/assign @ixdy

### CODE REVIEW

**Comment by krzyzacy** [comment]
Or maybe just need to strip out last entry?

File: scenarios/kubernetes_bazel.py [hunk 1]
```diff
@@ -83,12 +84,16 @@ def query(kind, selected_pkgs, changed_pkgs):
     if changed_pkgs:
         changes = 'set(%s)' % ' '.join(changed_pkgs)
 
-    return check_output(
+    for r in check_output(
```
Comment (ixdy) [inline scenarios/kubernetes_bazel.py]: `check_output(...).strip().split('\n')` maybe?

Comment (krzyzacy) [inline scenarios/kubernetes_bazel.py]: makes sense

**Pull request merged**